TODO.md

```diff
@@ -0,0 +1,5 @@
+# For upcoming release
```

Comment:
Yes, unless there is a reason to break it up into pieces, but it all seems logically connected to me, so I'd prefer to squash it to a single commit.